velox/expression/VectorUdfTypeSystem.h

```diff
@@ -337,6 +355,26 @@ struct VectorReader<Map<K, V>> {
     return !decoded_.isNullAt(offset);
   }
 
+  inline bool containsNulls(size_t index) const {
+    return decoded_.isNullAt(index) ||
+        (decodedKeys_.mayHaveNullsRecursive() &&
+         keyReader_.containsNulls(
+             offsets_[index], offsets_[index] + lengths_[index])) ||
```

Comment:
here you want to use decoded_.index(index)
ditto in other places.

Reply:
Great catch! I'll fix those and update the benchmark results in the other PR.

Reply:
Added test cases for these as well.